cirq-core/cirq/ops/linear_combinations_test.py

```diff
@@ -1156,6 +1156,18 @@ def test_pauli_sum_pow():
     for psum in [psum1, psum2, psum3, psum4]:
         assert cirq.approx_eq(psum**0, identity)
 
+    # tests for exponents greater than two for both even and odd 
+    psum5 = cirq.Z(q0) * cirq.Z(q1) + cirq.Z(q2) + cirq.Z(q3)
+    assert psum5 * psum5 * psum5 * psum5 * psum5 == psum5**5 
```

Comment:
nit - trailing blanks, please make sure `git diff --check master...HEAD` passes without complaints.

Reply:
Thanks, will do that from now on

Reply:
The change is not in effect, please see https://github.com/quantumlib/Cirq/pull/6019#issuecomment-1468971247

Reply:
I have removed the trailing blanks, but even when I add a white space and do git diff --check master...HEAD it doesn't show any complaints. I haven't dug into it much to see if I am missing something here.
For now I think checks on ./check/all should help